framework/src/consumption/lib/redshift/redshift-serverless/redshift-serverless-workgroup.ts

```diff
@@ -251,8 +253,7 @@ export class RedshiftServerlessWorkgroup extends TrackedConstruct implements ICo
    * then the Lambda function's security group would be added in the existing VPC endpoint's security group.
    * @returns `RedshiftData`
    */
-  public accessData(id: string, createVpcEndpoint?: boolean, existingInterfaceVPCEndpoint?: IInterfaceVpcEndpoint): RedshiftData {
-
+  private accessData(id: string, createVpcEndpoint?: boolean, existingInterfaceVPCEndpoint?: IInterfaceVpcEndpoint): RedshiftData {
```

Comment:
Should be public but marked as deprecated

Reply:
Pushed the change